cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/activity_search_facets_and_results.html

```diff
@@ -104,8 +102,7 @@ <h5>{{topic_dict[topic.title]}}</h5>
     </form>
 
 </aside>
-<section id="tdp-search-results" class="content_main content__flush-all-on-small content__flush-bottom results{% if not activities %} results__full{% endif %}" tabindex="-1">
-    <a class="u-visually-hidden" id="content_main"></a>
```

Comment:
@anselmbradford I'm unclear what purpose this line served, but without removing it the page layout was totally broken (the footer content got pulled up into the search results area). It seems like the first link in a `.content_main` block has some special styling which I didn't quite understand. Can you confirm that this change seems alright, given your understanding of the TDP frontend?

Reply:
Rather, I think this is (likely) vestigial from copying template code over from regs3k, where a `skip_nav_target` is set to `#content_main`. Removing this seems fine to me.